data/assets/scene/solarsystem/planets/earth/lagrange_points/l2/transforms.asset

```diff
@@ -0,0 +1,57 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms")
+local kernels = asset.require("./kernels")
+local coreKernels = asset.require("spice/core")
+
+
+
+local L2Position = {
+  Identifier = "L2Position",
+  Parent = transforms.SolarSystemBarycenter.Identifier,
+  Transform = {
+    Translation = {
+      Type = "SpiceTranslation",
+      Target = kernels.ID.L2,
+      Observer = coreKernels.ID.SolarSystemBarycenter
+    }
+  },
+  Tag = { "lagrange_points_earth", "lagrange_points_earth_l2" },
+  GUI = {
+    Name = "L2 Position",
+    Path = "/Solar System/Planets/Earth/Lagrange Points",
+    Hidden = true
+  }
+}
+
+-- This ref. frame co-revolves with L2 and is needed for a more intuitive trail of JWST in
+-- relation to L2
+local L2CoRevFrame = {
```

Comment:
Should we move this over to the JWST portion of the filesystem?

Reply:
Makes sense to me. @Roxeena

Reply:
In one way I agree, right now this is only used for the JWST, but this is a more general thing, and considering that I don't agree. For example, the GAIA trail might want to use this too to have a more intuitive L2 orbit trail than what we currently have. There are other spacecraft already at L2 and future spacecraft that will go there that will have a similar orbit around L2 that might want to use this frame as well. 

Perhaps the comment should be updated to reflect this general use case.

Reply:
I like the idea up updating the comment to make it clearer that it's not JWST specific. 

The JWST specific part could be moved to where this frame is being used instead, in the JWST-related asset :)

Reply:
Rephrased the comment to be a bit more general, but it still mentions JWST for easier understanding. Let me know if you think it looks ok